integration_tests/features/TransactionInfo.feature

```diff
@@ -0,0 +1,26 @@
+@merge-mining
+    Feature: Transaction Info
+
+    @critical
+        Scenario: Get Transaction Info
+            Given I have a seed node NODE
+            And I have 1 base nodes connected to all seed nodes
+            And I have wallet WALLET_A connected to all seed nodes
+            And I have wallet WALLET_B connected to all seed nodes
+            And I have a merge mining proxy PROXY connected to NODE and WALLET_A
+            When I merge mine 2 blocks via PROXY
+            Then all nodes are at height 2
+            And I send 1000000 tari from WALLET_A to one wallet WALLET_B at fee 100
+            Then Transaction status of last result from WALLET_A to WALLET_B is known to both wallets
+            When I merge mine 2 blocks via PROXY
```

Comment:
What's the reason for these extra steps?

Reply:
Good observation; was just to see if the transaction gets mined at the end, which is not the objective of the test.
Currently, it does not get detected mined, as here is an issue with the wallet communicating to the base node, although it was mined in block 3.
``` rust
2021-02-02 09:19:48.829478800 [wallet::transaction_service::protocols::broadcast_protocol] DEBUG Connecting to Base Node (Public Key: 5074a509cb27b35bec6733571206a26ca441f85b6589a809446549cc23fe3a41)
2021-02-02 09:19:50.481179100 [wallet::transaction_service::protocols::broadcast_protocol] INFO  Problem connecting to base node: ConnectionFailed: Failed to connect to peer within the maximum number of attempts for Transaction Broadcast Protocol (TxId: 1312964102198473045)
```
The transaction was mined in block 3 (See `Kernel 0`):
``` rust
2021-02-02 08:56:54.168588300 [c::bn::comms_interface::inbound_handler] TRACE Block: ----------------- Block -----------------
--- Header ---
Hash: cd294d5a61068478c9161e29a9b96f14be5b89fab43db5af5a32fecb2223ee4c
Version: 1
Block height: 3
Previous block hash: fbe051872f5252fd324061f86eb9b3b34e3870b451336a6b5459a22b84e13b1b
Timestamp: Tue, 02 Feb 2021 06:56:53 +0000
Merkle roots:
Outputs: 03050d04ec6bfc482e4656a38d1088705bb1475a08fa63a4393bb42d949cd92c (4006)
Range proofs: 7b768c320f90bd8af3410c984a4184afc2d95e0ca82673ad4e58b3b0b784f850
Kernels: 4db11a16e9f4f4a85d67f237b9a21834e8985762d437d31b6f6e3f305f5d94c9 (6)
Total offset: 6485277a7d5de9a888566793f1cab598862f864c6be57fb296a902a598b47809
Nonce: 0
Proof of work:
Mining algorithm: Monero
Pow data: MoneroBlockHeader: Major version: 14
Minor version: 14
Timestamp: 1612249013
Previous id: 0xa1ca…e8ea
Nonce: 0
 
RandomX vm key: 00570f482e581e1ea705c2f9516ba9b34b4faca861b1cd407e4534864280e677
Monero tx count: 1
Monero tx root: 81b85a87d8011edb66490a906ae1eb9fa9aae3e972038bb1e991d7997f29008c
Monero coinbase tx: Prefix: Version: 2
Unlock time: 764626
Extra field: Subfield: Tx public Key: 5ab25b2d7adab51dcb31a81d73c358e475a62fb1b4cad773b8a45bde88c0a492

Subfield: Nonce: 3c000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

Subfield: Merge mining: 0, 0x497f…8e82



RCT signature: Signature: RCT type: Null
Tx fee: 0





---  Body  ---
WARNING: Block body is not sorted.
--- Transaction Kernels ---
Kernel 0:
Fee: 3000 µT
Lock height: 0
Features: (empty)
Excess: c6eb9779654cb40f14e01837ccfb2f61df211985e169f3d6d5cec07d2931082f
Excess signature: {"public_nonce":"169942d7afc582e9d218b47649f8f6d58b93a8ea3390aed20d9ccca43f6e4e65","signature":"8ff7c37dc3d07e0e73f944f147c15e1f7dc80481029eda9e6d182a9489866b06"}

Kernel 1:
Fee: 0 µT
Lock height: 0
Features: COINBASE_KERNEL
Excess: 3a55f1ea7cc55ecbc8e958a66eca5b89356092b84b7a31e65d5031a40a03702a
Excess signature: {"public_nonce":"a81c633d23368882501b62451563221a801256f1e1bde6e33ae6c5ac0d971f65","signature":"7204032c55f74b5591bace2c4fa666e4032d469ab0a840916ec4bebce878d402"}

--- Inputs (1) ---
dc2b518fd44745f379cebdfc78360e9808f22cd258d2d4e87334c1b94f2e9050 [OutputFeatures { flags: COINBASE_OUTPUT, maturity: 2 }]
--- Outputs (3) ---
(1e51c2dfb9c2c675015ea8046f6e73e35cbc599baaf84cdf05a944c39d4d1103 [OutputFeatures { flags: (empty), maturity: 0 }] Proof: bcc937ba72111a4a..cfaeb701b405920f)(ec97c9f182f5f659abe83a186ba4da2a84831b691c0831db9514876a000b7012 [OutputFeatures { flags: (empty), maturity: 0 }] Proof: b018ca599caf0f67..ee2c3662239a860d)(6a1e91bc87b8b2ee8a1d362ea5ea29acb0fbd08cea2ee778f8ebfdafdab2d002 [OutputFeatures { flags: COINBASE_OUTPUT, maturity: 4 }] Proof: 78e87276700d6a49..acfa8db74ff54202)
```